devfile.yaml

```diff
@@ -10,7 +10,10 @@ components:
           name: nodejs
           protocol: http
           targetPort: 3000
-      memoryLimit: 1G
+      memoryLimit: '1Gi'
+      memoryRequest: '512Mi'
+      cpuLimit: '500m'
+      cpuRequest: '250m'
```

Comment:
in other places you have set 1 or 0.5; why not have 0.25 here?

Reply:
updated